Header header
uint32 point_number 		#point 1, 2 ,3...
geometry_msgs/Pose pose 	#how to relate EE back to original frame
motoman_sia20d_moveit_updated/trajPlotPointInfo[] pointInfo
